config/features/contact/core.entity_view_display.node.contact.token.yml

```diff
@@ -26,6 +26,11 @@ content:
     third_party_settings: {  }
     weight: 1
     region: content
+  content_moderation_control:
+    settings: {  }
+    third_party_settings: {  }
+    weight: -20
+    region: content
```

Comment:
Is this intentional? Seems like we wouldn't need this field in a token display...